src/mesh/analysis.rs

```diff
@@ -105,10 +105,9 @@ impl BoundingBox {
     }
 
     /// Check if the two bounding boxes intersect / share any portion of space
-
-    // https://math.stackexchange.com/questions/2651710/simplest-way-to-determine-if-two-3d-boxes-intersect
+    /// https://math.stackexchange.com/questions/2651710/simplest-way-to-determine-if-two-3d-boxes-intersect
```

Comment:
Again, no need to change this now, but note that these comments are markdown and that they do have a usual format they should be in (when in doubt, look at std documentation). This is how this actually renders:

![rendered](https://user-images.githubusercontent.com/1099245/71778561-dbb3d700-2faf-11ea-9686-ee68efc55911.png)